packages/@webex/plugin-meetings/src/metrics/index.ts

```diff
@@ -452,6 +453,29 @@ class Metrics {
     return this.generateErrorPayload(errorCode, showToUser, error.name.LOCUS_RESPONSE, err);
   }
 
+  /**
+   * Pareses webex api error.
+   *
+   * @param {object} err
+   * @param {boolean} showToUser
+   * @returns {any}
+   */
+  parseWebexApiError(err: any, showToUser: boolean) {
+    const serviceErrorCode = err?.body?.code;
+    const clientCodeError = WebexAPIServiceErrorCodes[serviceErrorCode];
+
+    return this.generateErrorPayload(clientCodeError, showToUser, error.name.OTHER, err);
```

Comment:
clientCodeError may be undefined - I think the code in generateErrorPayload() will handle that, but maybe better to not call it at all in that case or define the argument types in generateErrorPayload() to say that errorCode can be undefined?